### PR TITLE
Various small fixes and improvements

### PR DESCRIPTION
**Pull request by kosecki123**

### CODE REVIEW

**Comment by adibas03** [comment]
Can we clear all unneeded changes in this commit, so we could merge it
Indicated by `+`

File: contracts/DebtToken.sol [hunk 1]
```diff
@@ -16,25 +16,22 @@ contract DebtToken is ERC20Basic,MintableToken{
   Actual logic data
   */
   uint256 public decimals;
-  uint256 public dayLength = 86400;//Number of seconds in a day
-  uint256 public loanTerm;//Loan term in days
+  uint256 public requiredMaturityPeriod; //Loan term in days
```
Comment (adibas03) [inline contracts/DebtToken.sol]: +1

File: contracts/DebtToken.sol [hunk 2]
```diff
@@ -97,7 +93,7 @@ contract DebtToken is ERC20Basic,MintableToken{
     if(loanActivation == 0)
       return false;
     else
-      return now >= loanActivation.add( dayLength.mul(loanTerm) );
+      return now >= loanActivation.add(requiredMaturityPeriod.mul(1 days));
```
Comment (adibas03) [inline contracts/DebtToken.sol]: +1

File: contracts/DebtToken.sol [hunk 3]
```diff
@@ -107,7 +103,7 @@ contract DebtToken is ERC20Basic,MintableToken{
     if(!loanMature())
       return true;
     else
-      return !( now >= lastinterestCycle.add( interestCycleLength.mul(dayLength) ) );
+      return !(now >= lastinterestCycle.add(interestCycleLength.mul(1 days)));
```
Comment (adibas03) [inline contracts/DebtToken.sol]: +1

File: contracts/DebtToken.sol [hunk 4]
```diff
@@ -124,8 +120,8 @@ contract DebtToken is ERC20Basic,MintableToken{
       return (0,0);
     else{
       uint timeDiff = now.sub(lastinterestCycle);
-      _cycle = timeDiff.div(dayLength.mul(interestCycleLength) );
-      _coins = _cycle.mul( interestRate.mul(initialSupply) ).div(divisor);//Delayed division to avoid too early floor
+      _cycle = timeDiff.div(interestCycleLength.mul(1 days));
```
Comment (adibas03) [inline contracts/DebtToken.sol]: +1

File: contracts/DebtToken.sol [hunk 5]
```diff
@@ -124,8 +120,8 @@ contract DebtToken is ERC20Basic,MintableToken{
...
+      _coins = _cycle.mul( interestRate.mul(initialSupply) ).div(100);//Delayed division to avoid too early floor
```
Comment (adibas03) [inline contracts/DebtToken.sol]: +1

File: contracts/DebtToken.sol [hunk 6]
```diff
@@ -139,7 +135,7 @@ contract DebtToken is ERC20Basic,MintableToken{
     (interest_coins,interest_cycle) = calculateInterestDue();
     assert(interest_coins > 0 && interest_cycle > 0);
     totalInterestCycle =  totalInterestCycle.add(interest_cycle);
-    lastinterestCycle = lastinterestCycle.add( interest_cycle.mul( interestCycleLength.mul(dayLength) ) );
+    lastinterestCycle = lastinterestCycle.add( interest_cycle.mul( interestCycleLength.mul(1 days) ) );
```
Comment (adibas03) [inline contracts/DebtToken.sol]: +1

File: contracts/DebtToken.sol [hunk 7]
```diff
@@ -157,7 +153,7 @@ contract DebtToken is ERC20Basic,MintableToken{
     balances[owner] = balances[owner].sub(totalSupply);
     balances[msg.sender] = balances[msg.sender].add(totalSupply);
     loanActivation = now;  //store the time loan was activated
-    lastinterestCycle = now.add(dayLength.mul(loanTerm) ) ; //store the date interest matures
+    lastinterestCycle = now.add(requiredMaturityPeriod.mul(1 days)) ; //store the date interest matures
```
Comment (adibas03) [inline contracts/DebtToken.sol]: +1

**Comment by kosecki123** [comment]
This PR shall be closed. Most of the changes has been accumulated in `naming-convention` PR

**Comment by adibas03** [comment]
For reference :
 https://github.com/chronologic/debt-smart-contract/pull/11

**Pull request closed without merging**